### PR TITLE
[freeimage] Tiff targets for freeimage.

### PR DESCRIPTION
**Pull request by kobrineli**

### CODE REVIEW

File: projects/freeimage/README.md [hunk 1]
```diff
@@ -26,6 +26,24 @@ Run hybrid fuzzing:
 
     # sydr-fuzz -c load_from_memory.toml run
 
+Minimize corpus:
+
+    # sydr-fuzz -c load_from_memory.toml cmin
+
+Collect and report coverage:
```
Comment (SweetVishnya) [inline projects/freeimage/README.md:33]: There is no coverage binary

File: projects/freeimage/load_from_memory_tiff.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 ISP RAS
```
Comment (SweetVishnya) [inline projects/freeimage/load_from_memory_tiff.toml:1]: 2022

File: projects/freeimage/load_from_memory_tiff.toml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+args = "-l debug -n 1024 --app-timeout 60"
```
Comment (SweetVishnya) [inline projects/freeimage/load_from_memory_tiff.toml:18]: `-n 1024` -> `-s 90`

Comment (SweetVishnya) [reply projects/freeimage/load_from_memory_tiff.toml:18]: Is 60 app timeout enough?

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/freeimage/README.md (hunk 1 above)
Comment (kobrineli) [reply projects/freeimage/README.md]: done

File: projects/freeimage/load_from_memory_tiff.toml (hunk 2 above)
Comment (kobrineli) [reply projects/freeimage/load_from_memory_tiff.toml]: done

File: projects/freeimage/load_from_memory_tiff.toml (hunk 3 above)
Comment (kobrineli) [reply projects/freeimage/load_from_memory_tiff.toml]: done

File: projects/freeimage/README.md [hunk 4]
```diff
@@ -26,6 +26,20 @@ Run hybrid fuzzing:
 
     # sydr-fuzz -c load_from_memory.toml run
 
+Minimize corpus:
```
Comment (SweetVishnya) [inline projects/freeimage/README.md:29]: remove minimize corpus (stated twice)

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/freeimage/README.md (hunk 4 above)
Comment (kobrineli) [reply projects/freeimage/README.md]: done

**Pull request merged**